staging/src/k8s.io/client-go/examples/informer/README.md

```diff
@@ -0,0 +1,87 @@
+# Informer Example
+
+Informers provide a high-level API for creating custom controllers for Kubernetes resources.
+
+This particular example demonstrates:
+
+* How to write an Informer against a core resource type.
+* How to handle add, update and delete events.
+
+The [thirdparty-resources example](../third-party-resources) demonstrates how to use informers against a thirdparty resource.
+
+## Running
+
+To run the example outside the Kubernetes cluster you need to supply the path to a Kubernetes config file.
+
+```sh
+go run main.go -kubeconfig=$HOME/.kube/config -logtostderr
+```
+
+By default `glog` logs to files.
+Use the `-logtostderr` command line argument so that you can see the output on the console.
+
+## Running Inside a Kubernetes Cluster
+
+You can also run the example inside a Kubernetes cluster.
```

Comment:
Could we omit the instruction on how to start minikube? After all there are many other ways to get a kubernetes cluster. Instead just something like "Check out this [guide](https://kubernetes.io/docs/setup/) on how to setup a Kubernetes cluster".